packages/react/src/index.js

```diff
@@ -39,6 +39,7 @@ const Card = props => {
     setData,
     url,
     apiKey, // destructuring to avoid pass it
+    placeholderComponent: CardEmpty,
```

Comment:
not sure what could be the right naming for this:

- placeholderComponent
- skeletonComponent
- loadingComponent

I think `*Component` suffix is important to don't confuse it with a regular API parameter option

Reply:
Another structure for this sort of situation is a `components` prop, that receives an object where the keys are the component names in PascalCase (this is how `react-select` handles things).

```jsx
components={{ PlaceholderComponent: Foo }}